airflow/config_templates/config.yml

```
@@ -1502,6 +1502,13 @@
       type: string
       example: "dagrun_cleared,failed"
       default: ~
+    - name: enable_swagger_ui
+      description: |
+        Boolean for running SwaggerUI in the webserver.
+      version_added: 2.5.1
```

```suggestion
      version_added: 2.6.0
```
Should this be in config 'api' section, since swagger is about REST API?